db/migrate/20210220165936_add_essential_flags_to_feedback.rb

```diff
@@ -0,0 +1,5 @@
+class AddEssentialFlagsToFeedback < ActiveRecord::Migration[6.1]
+  def change
+    add_column :exercise_representations, :feedback_type, :tinyint, null: true
```

Comment:
We have a default for the feedback type, right? Did you prefer to keep it null instead of using a default value?

Reply:
I swung around with this. 

There might not be feedback on a representation (in fact that's the default state - we get a representation but no-one has added feedback to it) so I've kept it as null by default for that scenario. But I did also consider just giving it a default value as well though, which only gets used if there is feedback. Instead the `has_feedback?` method checks for this presence of this as part of it's calculation. 

In the UI, we will default it to `actionable`.